Src/VimCore/CommonOperations.fs

```diff
@@ -83,6 +84,60 @@ type internal CommonOperations
 
     member x.CaretLine = TextViewUtil.GetCaretLine _textView
 
+    /// The snapshot point in the buffer under the mouse cursor
+    member x.MousePoint =

```

Comment:
How does this differ from the typical way of looking at the caret point? Is it just applying all of the vim logic on top of it?

Reply:
It's not a caret point; it's a buffer point. What this API does is effectively translate the mouse position as supplied by the mouse device in pixel coordinates into a setting-sensitive virtual buffer point.

Reply:
I see now that I had done all of the work that made this API available to both command util and insert util, but then changed it so that insert util no longer needs it. So I could now move `x.MousePoint` back into command util where it was before.